src/wrapper/Plumed.h

```diff
@@ -3831,7 +3831,7 @@ void* plumed_attempt_dlopen(const char*path,int mode) {
       __PLUMED_FPRINTF(stderr,"+++ Allocation error +++\n");
       __PLUMED_WRAPPER_STD abort();
     }
-    __PLUMED_WRAPPER_STD strncpy(pathcopy,path,strlenpath+1);
+    __PLUMED_WRAPPER_STD strcpy(pathcopy,path);
```

Comment:
The ideal would be strdup, but it is not ansi C (actually is c23, from my understanding)

the +1 on the length with strncpy activates the warning about an overflow

by reading the manual strcpy copies the whole string plus the `\0`.